canonical-webteam/global-nav
# Append is-active class and update styles

## Done
Use the classList to add the class `is-active` to the link instead of replacing the classes. Limited the border top to just medium and large screens.

## QA
- Pull this branch and run `gulp build`
- Open the index.html
- Select one of the links in the global nav and add the `is-active` class. For example `global-nav__link is-active`.
- Check that the link turns orange and it has a border at the top
- Scale down to small screens and see that the link is just orange and has no border at the top
